2017/Day 2-2.py

```diff
@@ -0,0 +1,15 @@
+checksum = 0
+with open("Day 2 - input", "r") as file:
+    for line in file:
+        row = [int(i) for i in line.split('\t')]
+        for i in range(0,len(row)):
+            for second in row[i+1:]:
```

Comment:
You can use `itertools.combinations()` here to get all two number combinations. Then you can break the loop whenever `result` is not zero at the end, instead of processing the remaining combinations